src/corehost/build.cmd

```diff
@@ -0,0 +1,160 @@
+@if not defined _echo @echo off
+setlocal
+
+:SetupArgs
+:: Initialize the args that will be passed to cmake
+set __nativeWindowsDir=%~dp0Windows
+set __binDir=%~dp0..\..\bin
+set __rootDir=%~dp0..\..
+set __CMakeBinDir=""
+set __IntermediatesDir=""
+set __BuildArch=x64
+set __appContainer=""
+set __VCBuildArch=x86_amd64
+set CMAKE_BUILD_TYPE=Debug
+set "__LinkArgs= "
+set "__LinkLibraries= "
+
+:Arg_Loop
```

Comment:
This is a lot of logic in a .cmd file.  Can we port this to powershell instead, so our build logic doesn't need to have another language?